.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           command: |
             cd build
             make install
-            make appimage
+            make appimage || cat appimage.log
```

Comment:
👍

Reply:
Won't this mark the build as successful if AppImage creation is failed? I think it's fine if desired though.

Reply:
Perhaps `make appimage || cat appimage.log && false`

Reply:
> Perhaps `make appimage || cat appimage.log && false`

I think there should be a pair of parenthesis:  `make appimage || (cat appimage.log && false)`
See also: https://unix.stackexchange.com/questions/88850/

Reply:
I think if appimage creation fails, it will fail one line later in `cp ./lmms-*.AppImage /tmp/artifacts/`. I'll still add the `&& false` to make sure.

Reply:
> I think if appimage creation fails, it will fail one line later in `cp ./lmms-*.AppImage /tmp/artifacts/`.

True, I just missed that.